src/components/SearchBar.js

```diff
@@ -1,259 +1,292 @@
-import React, { useState } from 'react';
-import { TextField, Button, Typography } from '@mui/material';
-import SearchIcon from '@mui/icons-material/Search';
-import { DataGrid} from '@mui/x-data-grid';
-import {SAR_BACKEND_URL} from '../constants/urlConstants';
-import axios from 'axios';
-import "../css/SearchBar.css"
-import Loading from './Loading';
-import ExampleSearches from './ExampleSearches';
-import { columns} from '../constants/columns';
-import { permissionColumns } from '../constants/permissionColumns';
-import Link from '@mui/material/Link';
+import React, { useState } from "react";
+import { TextField, Button, Typography } from "@mui/material";
+import SearchIcon from "@mui/icons-material/Search";
+import { DataGrid } from "@mui/x-data-grid";
+import { SAR_BACKEND_URL } from "../constants/urlConstants";
+import axios from "axios";
+import "../css/SearchBar.css";
+import Loading from "./Loading";
+import ExampleSearches from "./ExampleSearches";
+import { columns } from "../constants/columns";
+import { permissionColumns } from "../constants/permissionColumns";
+import Link from "@mui/material/Link";
 
-// For the checkbox 
-import FormGroup from '@mui/material/FormGroup';
-import FormControlLabel from '@mui/material/FormControlLabel';
-import Checkbox from '@mui/material/Checkbox';
-import Stack from '@mui/material/Stack';
+// For the checkbox
+import FormGroup from "@mui/material/FormGroup";
+import FormControlLabel from "@mui/material/FormControlLabel";
+import Checkbox from "@mui/material/Checkbox";
+import Stack from "@mui/material/Stack";
 
 //Tooltip
-import InfoIcon from '@mui/icons-material/Info';
-import { Tooltip } from '@mui/material';
+import InfoIcon from "@mui/icons-material/Info";
+import { Tooltip } from "@mui/material";
 
+// zip
+let JSZip = require("jszip");
```

Comment:
I am getting the following error.

```
Failed to compile.

Module not found: Error: Can't resolve 'jszip' in '{directory}/sar-frontend/src/components'
ERROR in ./src/components/SearchBar.js 30:12-28
Module not found: Error: Can't resolve 'jszip' in '{directory}/sar-frontend/src/components'

webpack compiled with 1 error
```

Is the `jszip` package added in the package.json?